sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -612,16 +612,37 @@ Note, each resource must be in the same region as the endpoint passed in when in
 ```C# Snippet:QueryBatchMetrics
 string resourceId =
     "/subscriptions/<id>/resourceGroups/<rg-name>/providers/<source>/storageAccounts/<resource-name-1>";
-MetricsBatchQueryClient client = new MetricsBatchQueryClient(new Uri("https://metrics.monitor.azure.com/.default"), new DefaultAzureCredential());
-Response<MetricsBatchResult> metricsResultsResponse = await client.QueryBatchAsync(
-    resourceIds: new List<string> { resourceId },
+MetricsClient client = new MetricsClient(new Uri("https://metrics.monitor.azure.com/.default"), new DefaultAzureCredential());
```

Comment:
Reminder to update the client and method names in the text above. I assume you'll change that in your docs PR.